### DISCUSSION TITLE
怎么打出“攻略”？

### DISCUSSION THREAD

**Discussion by nmnmcc** [Mintimate/oh-my-rime#139] [Q&A] [answered]
gonglue不行，gongl可以

**Comment by Mintimate** [comment, marked as answer]
输入法里，除非模糊拼音，不然其实`ü`对应的是`v`。所以攻略，就是`gong lve`。

参考: https://zh.wikipedia.org/zh-hk/Topic:Uq9qprjyqnrd1wkq
